helpers/z_schema.js

```diff
@@ -49,27 +49,15 @@ z_schema.registerFormat('username', function (str) {
 });
 
 z_schema.registerFormat('hex', function (str) {
-	try {
-		Buffer.from(str, 'hex');
-	} catch (e) {
-		return false;
-	}
-
-	return true;
+	return /^[a-fA-F0-9]*$/.test(str);
```

Comment:
This validator is only for hex strings. We have different validators for `publicKey` and `signature` which check whether the length is equal to 64 and 128.
An empty string was valid previously, so I didn't change it.